website/versioned_docs/version-5.x/tab-based-navigation.md

```diff
@@ -0,0 +1,265 @@
+---
```

Comment:
we don't have a version-5.x folder

Reply:
I'm following the instructions on https://github.com/react-navigation/react-navigation.github.io/blob/source/.github/PULL_REQUEST_TEMPLATE.md to add versioned docs since this is for `5.x`. Should this file not to be add? Or should it be add to a different location?